cloudfoundry.md

```diff
@@ -130,6 +130,8 @@ cf set-env foia DATABASE_URL [value]
 cf set-env foia FOIA_ANALYTICS_ID [value]
 cf set-env foia FOIA_SECRET_SESSION_KEY [value]
 cf set-env foia DJANGO_SETTINGS_MODULE foia_hub.settings.dev
+cf set-env foia NEW_RELIC_LICENSE_KEY=[value]
+cf set-env foia NEW_RELIC_APP_NAME=[value]
```

Comment:
Makes sense. I guess the issue is if we want the New Relic app name to be static or changeable. Personally, I'm leaning towards having the app name set by an env variable because it's easy to setup and it would be useful for running a staging server/tests. We could also set the same New Relic app name for foia-a and foia-b. 

Thoughts?